packages/@okta/courage-for-signin-widget/package.json

```diff
@@ -4,11 +4,12 @@
   "description": "generates a Courage version for Sign-In Widget",
   "main": "src/CourageForSigninWidget.js",
   "scripts": {
-    "build": "scripts/build.sh"
+    "build": "webpack"
   },
   "devDependencies": {
-    "@okta/babel-plugin-handlebars-inline-precompile": "3.1.0-alpha.3586.gd86894f",
-    "@okta/courage": "4.6.0-alpha.3586.gd86894f",
+    "@okta/babel-plugin-handlebars-inline-precompile": "3.1.0-alpha.3660.g8faa6ac",
+    "@okta/eslint-plugin-okta-ui": "0.1.0-alpha.3660.g8faa6ac",
```

Comment:
what's `@okta/eslint-plugin-okta-ui` for?

Reply:
Thats where the `no-bare-templates` eslint rule is coming from. The same rule is run on courage